dist/index.d.ts

```diff
@@ -513,7 +564,6 @@ interface GPUCanvasConfiguration {
   viewFormats?: Iterable<GPUTextureFormat>;
   colorSpace?: GPUPredefinedColorSpace;
   compositingAlphaMode?: GPUCanvasCompositingAlphaMode;
```

Comment:
Let's keep this one for now since it was JUST removed from the spec.

```suggestion
  compositingAlphaMode?: GPUCanvasCompositingAlphaMode;
  /** @deprecated */
  size?: GPUExtent3D;
```